examples/tensorflow-comparison/comparison-training-multigpu.ipynb

```diff
@@ -0,0 +1,262 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5468cbd5",
+   "metadata": {},
+   "source": [
+    "# Training using Multiple GPUs\n",
+    "[Tensorflow](https://rapids.ai/) is a popular, powerful framework for deep learning used by data scientists across industries.\n",
+    "\n",
+    "In this example, you'll train Resnet50 architecture to identify different species of birds using multiple GPUs. This dataset constitutes 40,000+ birds and has been taken from [kaggle](https://www.kaggle.com/gpiosenka/100-bird-species).\n",
+    "\n",
+    "\n",
+    "First we import necessary libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad674cbb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "import keras\n",
+    "import time\n",
+    "\n",
+    "print(tf.__version__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b2f2fad5",
+   "metadata": {},
+   "source": [
+    "We will be using [Weights & Biases](https://wandb.ai/site) to monitor GPU performance. Users will need to set their own Saturn Cloud env credential for wandb. Check [here](https://saturncloud.io/docs/examples/python/weights-and-biases/qs-wandb/) on more information on creating and connecting W&B account to Saturn Cloud. First, we log into Weights & Biases. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7070eca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import wandb\n",
+    "from wandb.keras import WandbCallback\n",
+    "\n",
+    "wandb.login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f301ed9",
+   "metadata": {},
+   "source": [
+    "### Extracting Data\n",
+    "The dataset originally had 285 classes. We have taken subset of this data which has 61 classes . The data is stored in AWS S3.The first time you run this job, you'll need to download the training and test data which will be saved at path `dataset/birds/`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4738937d-656e-41dd-9ce8-7ac72cea9054",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import s3fs\n",
+    "\n",
+    "s3 = s3fs.S3FileSystem(anon=True)\n",
+    "_ = s3.get(\n",
+    "    rpath=\"s3://saturn-public-data/100-bird-species/100-bird-species/*/*/*.jpg\",\n",
+    "    lpath=\"dataset/birds/\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a3b0fc33",
+   "metadata": {},
+   "source": [
+    "Run the code below to ensure that TensorFlow just absorbs memory as needed, instead of absorbing all the RAM it has access to on your GPUs."
```

Comment:
```suggestion
    "Run the code below to ensure that TensorFlow just uses memory as needed, instead of using all the RAM it has access to on your GPUs."
```